operators/src/source/ogr_source.rs

```diff
@@ -715,17 +579,23 @@ where
         feature_iterator: &mut Peekable<FeatureIterator<'_>>,
         feature_collection_builder: FeatureCollectionBuilder<G>,
         dataset_information: &OgrSourceDataset,
-        default_geometry: &Option<G>,
         data_types: &HashMap<String, FeatureDataType>,
         query_rectangle: &QueryRectangle,
         time_extractor: &dyn Fn(&Feature) -> Result<TimeInterval>,
         chunk_byte_size: usize,
     ) -> Result<FeatureCollection<G>> {
         let mut builder = feature_collection_builder.finish_header();
 
+        // TODO: cleaner way of handling the default geometry that fails earlier on wrongt data type and avoids cloning
```

Comment:
>ChristianBeilschmidt commented on 18.01.2021
You deleted that previously

I meant a type safe way `enum { Skip, Error, Keep (DefaultGeometry)}`